### PR TITLE
Db 497 consolidate setup

### PR DESCRIPTION
**Pull request by jworcestBAH**
This branch consolidates the db setup for validation, staging, and job queue to be consistent with other DBs, and refactors initialize scripts to work correctly and include a -d option for deploying on AWS.

### CODE REVIEW

File: dataactvalidator/filestreaming/schemaLoader.py [hunk 1]
```diff
@@ -81,16 +84,19 @@ def loadCrossRules(filename):
                 fileId = validationDb.getFileId(record["file"])
                 validationDb.addMultiFieldRule(fileId,record["rule_type"],record["rule_text_one"],record["rule_text_two"],record["description"],record["rule_label"],record["rule_timing"])
 
+    @classmethod
```
Comment (bsweger) [inline dataactvalidator/filestreaming/schemaLoader.py]: I really like this revamp of the field and rule file loads. Do you think it makes sense to take it one step further and add top-level path of the broker backend and the path to the validator conifg in `dataactcore/config.py` so we don't have to worry about constructing that path down here?

**Comment by bsweger** [comment]
@jworcestBAH done reviewing!

File: dataactvalidator/filestreaming/schemaLoader.py (hunk 1 above)
Comment (jworcestBAH) [inline dataactvalidator/filestreaming/schemaLoader.py]: CONFIG_BROKER['path'] now holds the path to the base directory for the install.

**Comment by jworcestBAH** [comment]
Updated path handling in config.py to use abspath(__file__) throughout

**Comment by bsweger** [comment]
Looks good--thanks for adding the broker path to the config file. That will come in handy!

**Pull request merged**